frontend/appsettings.Development.json

```diff
@@ -8,6 +8,6 @@
   },
   "MyAppSettings": {
     "ApiUrl": "https://localhost:7097",
-    "OpenAIKey": "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"
+    "OpenAIKey": "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"
```

Comment:
!!! nooit API keys in de repo zetten, appsettings.Development.json stond ook in de .gitignore om die reden. Gebruik alternatively de methoden die hier beschreven staan: [Configuration in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/?view=aspnetcore-6.0)

Reply:
vandaar dat openAI ze ook blijft blokkeren, veel bedrijven scannen als het goed is voor compromised API keys